src/controller.rs

```diff
@@ -169,7 +169,7 @@ fn new_token_response(app_data: &AppData, audience: &str, grant_type: GrantType)
         custom_claims,
     );
 
-    let user_info: UserInfo = UserInfo::new(app_data.config(), audience.to_string());
+    let user_info: UserInfo = UserInfo::new(app_data.config(), CLIENT_ID_VALUE.to_string());
```

Comment:
Why this change? I think audience was the correct value

Reply:
According to [auth0 doc](https://auth0.com/docs/secure/tokens/id-tokens/validate-id-tokens) the audience should be the `client_id`. Also my `openconnectid` lib expects that to be the `client_id` [audience check](https://github.com/ramosbugs/openidconnect-rs/blob/9642f2252cc8c66a6469fbcc76a321ec5a123fe6/src/verification.rs#L300)

Reply:
Peraps, we should also implement the nonce exchange later on.

Reply:
Luca is right, in id_token "aud" field is valued with the client id of the application. In access_token refers to audience instead.

I see no test broken due to this change. I'd add a test on id_token like the one "generate_access_token_with_custom_claims"

Reply:
I changed the existing test to include checks for the returned `id_token`. I changed that one because the `id_token` is returned by the same tested function. Tell me if you like this approach :wink:

Reply:
lgtm